code/app.js

```diff
@@ -0,0 +1,39 @@
+window.addEventListener('load', ()=> {
+    let long;
+    let lat;
+    let tempDescription = document.querySelector('.temp-description');
+    let tempDegree = document.querySelector('.temp-degree');
+    let locationTimezone = document.querySelector('.location-timezone');
+
+    if(navigator.geolocation){
+        navigator.geolocation.getCurrentPosition(position =>{
+            long = position.coords.longitude;
+            lat = position.coords.latitude;
+
+            const proxy = "https://cors-anywhere.herokuapp.com/";
+
+            const api = `${proxy}https://api.darksky.net/forecast/c48561e642e430bdaa676adbd2ace2e7/${lat},${long}`;

```

Comment:
You probably shouldn't commit your API key to Github. Now anyone can get it! 🗡 

A common solution to this problem is to make "secrets" a configurable part of the "environment". See https://12factor.net/config.